client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -5,24 +5,24 @@
             <div class="row d-md-flex">
                 <div class="col-sm-6">
                     <div class="oui-tile mb-5 h-100">
-                        <h3 class="oui-tile__title" data-i18n-static="domain_dashboard_general_informations"></h3>
+                        <h3 class="oui-header_4 oui-tile__title" data-i18n-static="domain_dashboard_general_informations"></h3>
                         <div class="oui-tile__body">
                             <ul class="list-unstyled">
                                 <!-- OFFER -->
                                 <li class="oui-tile__item" data-ng-if="!ctrlTabGeneralInformations.isAllDom">
-                                    <div class="oui-tile__definition">
-                                        <strong data-i18n-static="domain_dashboard_offer"></strong>
-                                        <div class="text-capitalize"
+                                    <div class="oui-tile__term">
+                                        <div data-i18n-static="domain_dashboard_offer"></div>
+                                        <div class="oui-tile__description"
                                              data-ng-bind="ctrlTabGeneralInformations.domain.offer"></div>
                                     </div>
                                 </li>
                                 <!-- OFFER ALLDOM -->
                                 <li class="oui-tile__item"
                                     data-ng-if="ctrlTabGeneralInformations.isAllDom && ctrlTabGeneralInformations.allDom">
-                                    <div class="oui-tile__definition">
-                                        <strong
-                                            data-ng-bind="::tr('domain_dashboard_type_alldom_' + ctrlTabGeneralInformations.constructor.parseType(ctrlTabGeneralInformations.allDom.type))"></strong>
-                                        <ul class="list-unstyled">
+                                    <div class="oui-tile__term">
+                                        <div
```

Comment:
Add class `font-weight-bold` if needed :)

Reply:
regarding the new guideline, text on bold is not necessary :)

Reply:
"new"